src/aind_dynamic_foraging_basic_analysis/metrics/trial_metrics.py

```diff
@@ -0,0 +1,61 @@
+"""
+    Tools for computing trial by trial metrics
+    df_trials = compute_all_trial_metrics(nwb)
+
+"""
+
+import numpy as np
+
+# TODO, we might want to make these parameters metric specific
+WIN_DUR = 15
+MIN_EVENTS = 2
+
+
+def compute_all_trial_metrics(nwb):
```

Comment:
I have more trial-based metrics [here](https://github.com/AllenNeuralDynamics/aind-foraging-behavior-bonsai-basic/blob/5bd0db90bf9182a969bcb3c52af77c8bb9a36f19/code/process_nwbs.py#L61) that can be migrated to this function.

Most of my metrics are not binary variables, so they may not need smoothing.

Reply:
Tracking them in this issue: https://github.com/AllenNeuralDynamics/aind-dynamic-foraging-basic-analysis/issues/27